nixos/modules/virtualisation/hyperv-guest.nix

```
@@ -10,6 +10,9 @@ in {
     virtualisation.hypervGuest = {
       enable = mkEnableOption (lib.mdDoc "Hyper-V Guest Support");
 
+      enhancedSession =
```

```suggestion
      enhancedSession.enable =
```
Should add `.enable`. This also makes @MatthewCash [suggested change](https://github.com/NixOS/nixpkgs/pull/300418/files#r1643678489) work